pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -614,6 +614,9 @@ type Outputs struct {
 
 	// Result holds the result (stdout) of a script template
 	Result *string `json:"result,omitempty" protobuf:"bytes,3,opt,name=result"`
+
+	// Holds the exit code of a script template
```

Comment:
It's convention that the doc string begins with the name of the field it references
```suggestion
	// ExitCode holds the exit code of a script template
```

Reply:
Ok, is that specific to Golang ?